slave/buildslave/monkeypatches/bug4881.py

```diff
@@ -189,7 +191,7 @@ def _resourceFDImplementation(self):
         # to close
         if maxfds > 1024:
             maxfds = 1024
-        return xrange(maxfds)
+        return list(range(maxfds))
```

Comment:
I might have interpreted the documentation wrong. I'll double check and then update it to be the correct type of ragnge